base/hsocket.h

```diff
@@ -51,23 +51,23 @@ typedef union {
     struct sockaddr     sa;
     struct sockaddr_in  sin;
     struct sockaddr_in6 sin6;
-} sockaddr_un;
```

Comment:
I believe the "un" here means "union", however, in <sys/un.h>, there is a `sockaddr_un` for Unix domains sockets where "un" means "Unix". On Solaris, <sys/un.h> is included by <sys/socket.h>, resulting in compilation error.

This issue also concerns Linux or BSD if libhv's going to support Unix domain sockets in the future. This conflict is not specific to Solaris, for `sockaddr_un` conforms to IEEE Std 1003.1-2008, and platforms like Linux and BSD also use that name.

Renaming to `sockaddr_u` is recommended.